clang/lib/CIR/CodeGen/CIRGenFunction.cpp

```diff
@@ -960,6 +960,29 @@ static bool matchesStlAllocatorFn(const Decl *D, const ASTContext &Ctx) {
   return true;
 }
 
+// TODO: this should live in `buildFunctionProlog
+/// An argument came in as a promoted argument; demote it back to its
```

Comment:
There's a mismatch in comment style `//` vs `///` and a missing backtick after buildFunctionProlog